tdrs-frontend/src/components/UploadReport/UploadReport.jsx

```diff
@@ -38,12 +39,8 @@ function UploadReport({ handleCancel, header, stt }) {
     headerRef.current.focus()
   }, [])
 
-  useEffect(() => {
-    dispatch(getAvailableFileList({ year: selectedYear }))
-  }, [dispatch, selectedYear])
-
-  const filteredFiles = files.filter((file) => file.fileName)
-  const uploadedSections = filteredFiles
+  const uploadedFiles = files.filter((file) => file.fileName && !file.id)
```

Comment:
Only upload files for sections that have a defined file name and do not yet have the ID, which indicates they have not been submitted yet.